src/app/hooks/lending/useGetLendingHistory.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import axios from 'axios';
+import { selectWalletProvider } from 'app/containers/WalletProvider/selectors';
+import { subgraphWrapperUrl } from 'utils/classifiers';
+import { LendingPool } from 'utils/models/lending-pool';
+import { getLendingContract } from 'utils/blockchain/contract-helpers';
+import { LoanTokenGraphItem } from 'app/components/LoanTokenGraphs/types';
+
+export function useGetLendingHistory(pool: LendingPool) {
+  const [data, setData] = useState<LoanTokenGraphItem[]>([]);
+  const { chainId } = useSelector(selectWalletProvider);
+  const poolAddress = getLendingContract(pool.getAsset()).address;
+
+  useEffect(() => {
+    if (chainId !== undefined) {
+      axios
+        .get(`${subgraphWrapperUrl[chainId]}/lendingApy/${poolAddress}`)
+        .then(res => {
+          setData(res.data.slice(-36)); //last 6 days of data in 4hr chunks
```

Comment:
`6*4 = 24` if I am not mistaken so either this comment or value doesn't make sense :D Which one of them is it?

Reply:
in one day (24hr) we have 6 points with 4hr interval, 24/4 = 6,
6*6 = 36 items to get last 6 days with 4hr chunks